jumanji/environments/routing/cleaner/generator_test.py

```diff
@@ -27,20 +27,23 @@ class TestRandomGenerator:
     HEIGHT = 7
     NUM_AGENTS = 3
 
-    @pytest.fixture
+    @pytest.fixture()
```

Comment:
Any reason why you have added an empty parenthesis here?

Reply:
No reason, just a new autoformatter I have and I forgot to disable this :unamused: Should be fixed now